lib/glific/third_party/bigquery/bigquery_schema.ex

```diff
@@ -1811,6 +1811,211 @@ defmodule Glific.BigQuery.Schema do
     ]
   end
 
+  @doc """
+  Schema for registrations table
+  """
+  @spec registrations_schema :: list()
+  def registrations_schema do
+    [
+      %{
+        description: "Unique ID generated for each registrations",
+        name: "id",
+        type: "INTEGER",
+        mode: "REQUIRED"
+      },
+      %{
+        description: "Unique UUID for the row (allows us to delete duplicates)",
+        name: "bq_uuid",
+        type: "STRING",
+        mode: "NULLABLE"
+      },
+      %{
+        description: "Time when the record entry was made on bigquery",
```

Comment:
I see schema for registration but where is this table being populated from in codebase? And We discussed to skip it for now and send data directly to notion